samples/Datadog Sample/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -28,6 +28,7 @@ android {
 }
 // Android Resolver Exclusions End
 android {
+    namespace "com.datadoghq.unityExample"
```

Comment:
maybe `com.datadog.unity.example`? at least we are already using `com.datadog.android` for Android SDK and `com.datadog.kmp` for KMP.

Reply:
note that Android SDK / KMP SDK are using `com.datadog`, not `com.datadoghq`, but probably it doesn't really matter.